CONTRIBUTING.rst

```diff
@@ -65,7 +65,7 @@ the ``URL`` will need to include both the PAT and username, e.g.
 JavaScript setup
 ================
 
-For working on the front-end, you need at least Node.js 14 and npm 7
+For working on the front-end, you need at least Node.js 16.6.0 and npm 7
```

Comment:
I'd drop the reference to the version number, otherwise we'll need to play maintenance catch-up every time it changes:

```suggestion
For working on the front-end, you need the latest versions of Node.js and npm
```